### DISCUSSION TITLE
Asus Merlin обход замедления YouTube

### DISCUSSION THREAD

**Discussion by Saints1nneR** [bol-van/zapret#285] [Q&A]
Доброго дня всем. А есть инструкция как сделать все это но на роутере ASUS с кастомом merlin?

**Comment by Karpoffd** [comment]
Поддерживаю тему, у меня почти получилось, но крашится на запуске, не хваетает NFQUEUE моему AC68U, может кто подскажет как поставить или пересобрать прошивку

**Reply by Sevinfolds** [reply]
Здравствуйте! Подскажите пожалуйста как пытались установить? На своём 88м хочу попробовать.

**Reply by Eric-1dev** [reply]
Надо установить xtables-addons_legacy. Потом можно загрузить модуль через

insmod /lib/modules/2.6.36.4brcmarm/kernel/net/netfilter/xt_NFQUEUE.ko (у вас путь может отличаться)

Подсмотрел тут https://habr.com/ru/articles/834826/comments/#comment_27189238

После этого перестало ругаться на строку типа
iptables -t mangle -I PREROUTING -i eth0 -p 145 -m u32 --u32 "0>>22&0x3C@0&0xFFFF=16" -j NFQUEUE --queue-num 300 --queue-bypass

Дальше пока не игрался

**Reply by PaulPoloziouk** [reply]
> Дальше пока не игрался

Получилось срочно доделать? Ситуация крайне плохая. На Merlin НЕТ стабильных и легко устанавливаемых сборок Zapret. Всё только с костылями, запуском 100500 скриптов с огромным количеством параметров и нюансов.

**Comment by Sevinfolds** [comment]
> Доброго дня всем. А есть инструкция как сделать все это но на роутере ASUS с кастомом merlin?

Поддерживаю, AC88U Merlin :)

**Reply by IndeecFOX** [reply]
Есть скрипт для простого развёртывания zapret. Merlin поддержка недавно была добавлена, но проверялась только с одним юзером на ax58u. При ошибках в развёртвывании писать в чат ТГ скрипта, он на гите указан
https://github.com/IndeecFOX/zapret4rocket

**Comment by ElectroBandit** [comment]
Поддерживаю. Тоже имею в разных локациях несколько роутеров Asus с прошивками Merlin + Entware. Но опыта и знаний не хватает для того, чтобы развернуть zapret. Есть инструкции для кинетиков и OpenWRT, но у асусов свои нюансы.

**Reply by IndeecFOX** [reply]
Есть скрипт для простого развёртывания zapret. Merlin поддержка недавно была добавлена, но проверялась только с одним юзером на ax58u. При ошибках в развёртвывании писать в чат ТГ скрипта, он на гите указан
https://github.com/IndeecFOX/zapret4rocket

**Comment by Elranon** [comment]
Добрый день
присоединяюсь к вопросу.... можно ли сделать гайд по устнановке на ASUSWRT(сток) или Merlin? роутер AX58U

**Reply by IndeecFOX** [reply]
Есть скрипт для простого развёртывания zapret. Merlin поддержка недавно была добавлена, но проверялась только с одним юзером на ax58u. При ошибках в развёртвывании писать в чат ТГ скрипта, он на гите указан
https://github.com/IndeecFOX/zapret4rocket

**Comment by bugsauce24** [comment]
присоединяюсь, но меня интересует роутер asus rt-ac68u с прошивкой Merlin

**Reply by IndeecFOX** [reply]
Есть скрипт для простого развёртывания zapret. Merlin поддержка недавно была добавлена, но проверялась только с одним юзером на ax58u. При ошибках в развёртвывании писать в чат ТГ скрипта, он на гите указан
https://github.com/IndeecFOX/zapret4rocket

**Comment by PaulPoloziouk** [comment]
@bugsauce24  Удалось срочно решить проблему? Что в итоге использовали?

**Reply by bugsauce24** [reply]
нет, не удалось, пришлось купить другой роутер, а асус закинуть в кладовку.
Мерлин у меня установился, но он не поддерживает wifi, поэтому роутер становится вполовину бесполезным.

**Reply by PaulPoloziouk** [reply]
@bugsauce24  Понял. Получается, что Мерлион это ересь. Эх... Ужасно расстроился из-за этого.
Из тех роутеров, что более-менее свежие и с поддержкой WiFi7 ни на один не ставится OpenWRT.
Какой роутер с поддержкой быстрого WiFi7 Вы срочно купили?

**Reply by numbereleven-a** [reply]
> @bugsauce24 Понял. Получается, что Мерлион это ересь. Эх... Ужасно расстроился из-за этого. Из тех роутеров, что более-менее свежие и с поддержкой WiFi7 ни на один не ставится OpenWRT. Какой роутер с поддержкой быстрого WiFi7 Вы срочно купили?

их не так много https://toh.openwrt.org/?view=all
SmartRGSDG-8733A
SmartRGSDG-8733
Arcadyan / Astoria - Mozart

**Reply by bol-van** [reply]
Действительно ли имеется смысл в столь быстром wifi ? Гигабитный тариф ? Необходимость качать файлики по локалке ? Девайсы-клиенты тоже поддерживают ?
На крайняк wifi7 ставится как точка доступа, а роутер выполняется на другой железке. На китайском mini pc, например.

**Reply by PaulPoloziouk** [reply]
Да, необходимость качать быстро большие объёмы есть. Я с ноутбука дома заканчиваю в сеть на работе большие файлы размером по 2-6 Гб. Качаются по WiF6 ооочень долго. Все упирается именно в WiFi6. Тариф от провайдера хороший и быстрый. Роутер Anus RT-AX88U явно устарел. Я его покупал лет 6 назад. Сейчас уже пора покупать явно на базе того, куда можно поставить OpenWRT. Например, быстрый NetGear с поддержкой  WiFi 7. Но он стоит более 850 долларов за рубежом, а у нас так и вовсе нигде не продаётся.

**Reply by bol-van** [reply]
При таких бюджетах можно себе позволить раскошелиться на миниписюк с алика в пределах 15K

**Reply by electrifying** [reply]
А есть ли сейчас ноутбуки с поддержкой wifi 7? Нашел в днс самый дешевый ноут у которого есть wifi 7 стоит 100 тысяч рублей. Разве сейчас wifi 6 не хватит для гигабитного интернета? Или это редкий тариф и провайдер кто начинает давать 2.5 гигабита и тут без wifi 7 не обойтись?

**Reply by bol-van** [reply]
wifi в ноутах не намертво в плату впаян, поэтому считать wifi принадлежностью ноута нельзя.
можно купить отдельно под M2 слот нужный адаптер и заменить
кстати, часто есть и возможность установки wifi карточек в minipc
если найти карточку с поддержкой работы в режиме точки доступа и с поддержкой Linux, то вот вам и роутер из миниПЦ
без 850 баксов

**Reply by bol-van** [reply]
Что-то вроде такого
https://aliexpress.ru/item/1005003378019857.html
А тут сразу выведены разьемы под антены
https://aliexpress.ru/item/1005008305014192.html

**Reply by PaulPoloziouk** [reply]
@bol-van 
Есть ли более быстрый роутер? У этого роутера плохое охлаждение, медленный процессор Церлеррон.

**Reply by bol-van** [reply]
Эти железяки переплюнут любой самый дорогой домашний роутер.
Особенно N100.
Потребляемая мощность в пределах 10 вт, охлаждение за счет корпуса.
У меня такой работает. Средняя температура CPU 40-45 градусов

**Reply by PaulPoloziouk** [reply]
@bol-van легендарный обзорщик роутеров Крашер (@CrashX), сайт https://urouter.ru  уже неоднократно писал, что все эти самодельные балалайки с АлиЭкспресс не поддерживают полноценной работы с WiFi7, а реализуют лишь костыльные решения. Чудес-то не бывает. Дёшево хорошо не бывает.
Так что срочно буду искать более качественные и быстрые варианты. Может быть Вы подскажете, что бы Вы сейчас купили из роутеров, имея 900 долларов США и почему? Я всё же ориентируюсь на большую тройку (Asus, D-Link, NetGear) ввиду их полного соблюдения всех необходимых нюансов касательно работы с сетями. В том числе и по поводу безопасности, отказоустойчивости и надёжности работы.

**Reply by bol-van** [reply]
Здесь мы говорим не о балалайках, а о полноценном PC.
Полноценность его как точки доступа определяет wifi модуль. Возможно, еще антены.
Любой домашний роутер - это какая-то слабая система на linux + wifi адаптер + антены.
В данном случае это более сильная система.

**Reply by bol-van** [reply]
Я точно не стал бы покупать топовые железяки за 80 рублей и использовал бы описанное мной решение.
Нашел бы несколько вариантов wifi 7 карточек. Покопался бы на форумах, в гугле. Поспрашивал бы есть ли опыт использования их с hostapd и каков он.

**Reply by bugsauce24** [reply]
> @bugsauce24 Понял. Получается, что Мерлион это ересь. Эх... Ужасно расстроился из-за этого. Из тех роутеров, что более-менее свежие и с поддержкой WiFi7 ни на один не ставится OpenWRT. Какой роутер с поддержкой быстрого WiFi7 Вы срочно купили?

Я купил роутер AX3000T от Xiaomi. На него всё легко установилось и прекрасно работает. По цене около 3000р вышло.
На нём нет 7 wifi, только 6.
